hoodie-common/src/main/java/com/uber/hoodie/common/table/log/block/HoodieAvroDataBlock.java

```diff
@@ -72,7 +78,7 @@ public Schema getSchema() {
     DataOutputStream output = new DataOutputStream(baos);
 
     // 1. Write out metadata
-    if (super.getLogMetadata() != null) {
+    if(super.getLogMetadata() != null) {
```

Comment:
same revert formatting.. (might want to double check your save action/code style setup)